config/init.lua

```diff
@@ -104,6 +105,10 @@ for _, key in pairs(keys) do
     config.register_key(key)
 end
 
+function cleanup_background()
+  os.execute("pkill way-cooler-bg")
+end
+
 
 -- Execute some code after Way Cooler is finished initializing
 function way_cooler_init()
```

Comment:
I still don't understand why this is here. Is this supposed to happen post-wlc-startup? I think it's definitely useful to have something like that in place because there are definitely race conditions as we load the init file and wlc concurrently, so if users want to i.e. open a program on startup they might have to wait until after the compositor environment exists or the program would crash. But in order to do this we have to send the Lua thread some kind of `COMPOSITOR_READY` message in the wlc callback. In that case, we should have a `way_cooler.on_compositor_is_ready` callback that the Lua session calls explicitly when it has received a message to do so. That might be better off in a different PR, especially if we want `compositor_terminating` to match.